<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>area_trapezoid_l_bases_h_Window</class>
 <widget class="QMainWindow" name="area_trapezoid_l_bases_h_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="area_trapezoid_l_bases_h_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вычисление площади трапеции, если известны длины оснований и высота.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="result_frame_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Результат</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="result_text_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../formula8.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="input_frame_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="a_input">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="a_label">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>a   </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="b_label">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>b   </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="b_input">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="h_label">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="h_input">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="area_trapezoid_l_bases_h_result_button">
       <property name="font">
        <font>
         <family>Source Code Pro Medium</family>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
